### DISCUSSION TITLE
Is the Text block supposed to be edited and rendered as plain text?

### DISCUSSION THREAD

**Discussion by horacioj** [PiranhaCMS/piranha.core#1549] [Q&A] [answered]
Is the Text block supposed to be edited and rendered as plain text? I ask because it acts as an HTML editor, and it is rendered as HTML. The only difference I see with the Content block is that the Text block doesn't include the editor's toolbar. However, the content is the same (HTML).

**Comment by tidyui** [comment]
The textblock is intended for unformatted text. It is rendered in the manager as a content editable div.

**Reply by horacioj** [reply]
If you type text, hit enter, type text, hit enter, etc. the content is saved as html with `<p>` for each time you hit enter. Furthermore, if you paste HTML, it is saved as is. In the example project,  the block is rendered as html.Raw, which makes it render as html.

Is this the intended behavior? Or should the content have been saved as plain text without html tags?

**Reply by horacioj** [reply]
also, this block isn't properly collapsed in the manager (collapsing each block on the page, from top to bottom), see attached examples.

### Typed (plain) text

![text-01](https://user-images.githubusercontent.com/984821/111306799-6d4cb900-8637-11eb-9d46-594cf623576b.jpg)

![text-02](https://user-images.githubusercontent.com/984821/111306797-6cb42280-8637-11eb-93dc-1c6caa834850.jpg)

### Pasted html

![text-03_pasted html](https://user-images.githubusercontent.com/984821/111306796-6cb42280-8637-11eb-8836-f400a47e422a.jpg)

![text-03_pasted html-code](https://user-images.githubusercontent.com/984821/111306792-6b82f580-8637-11eb-8aa7-23d775006f2f.jpg)

### Collapsing block from top to bottom

![text-03_pasted html - collapsed](https://user-images.githubusercontent.com/984821/111306801-6d4cb900-8637-11eb-9fe2-9bd145c37b65.jpg)

**Comment by tidyui** [comment, marked as answer]
This will be released in version `9.1`, see issue #1566
